maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gitexe/src/main/java/org/apache/maven/scm/provider/git/gitexe/command/GitCommandLineUtils.java

```diff
@@ -90,28 +95,36 @@ public static void addTarget( Commandline cl, List<File> files )
     }
 
    /**
-    *
+    * Use this only for command not requiring environment variables (i.e. local commands).
```

Comment:
commands

Reply:
Fixed in https://github.com/apache/maven-scm/pull/159/commits/1a3cfe51b14204bbef36b48f672af3f72df8a3eb.